examples/02-tips-n-tricks/00-example-template.py

```diff
@@ -20,25 +20,18 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-"""
+'''
 .. _ref_how_to_add_an_example_reference_key:
 
+============================
 Adding a New Gallery Example
```

Comment:
```suggestion
Adding a new gallery example
```
We should always be using sentence case for titles.